src/lib/support/DefaultStorageKeyAllocator.h

```diff
@@ -170,6 +170,14 @@ class DefaultStorageKeyAllocator
         return StorageKeyName::Formatted("g/a/%x/%" PRIx32 "/%" PRIx32, endpointId, clusterId, attributeId);
     }
 
+    // Returns the key for Safely stored attributes.
+    static StorageKeyName SafeAttributeValue(EndpointId endpointId, ClusterId clusterId, AttributeId attributeId)
+    {
+        // Needs at most 26 chars: 6 for "s/a///", 4 for the endpoint id, 8 each
+        // for the cluster and attribute ids.
+        return StorageKeyName::Formatted("s/a/%x/%" PRIx32 "/%" PRIx32, endpointId, clusterId, attributeId);
```

Comment:
```suggestion
        return StorageKeyName::Formatted("g/sa/%x/%" PRIx32 "/%" PRIx32, endpointId, clusterId, attributeId);
```

Reply:
This is very important: the key _has_ to start with "g/".